nixos/modules/services/networking/consul.nix

```diff
@@ -240,7 +230,8 @@ in
               --alert-addr=${cfg.alerts.listenAddr} \
               --consul-addr=${cfg.alerts.consulAddr} \
               ${optionalString cfg.alerts.watchChecks "--watch-checks"} \
-              ${optionalString cfg.alerts.watchEvents "--watch-events"}
+              ${optionalString cfg.alerts.watchEvents "--watch-events"} \
+              --log-level=warn
```

Comment:
it just fails to start without explicit ```--log-level=```.
the previous version (before the PR)  too.